eo-runtime/src/test/java/org/eolang/MainTest.java

```diff
@@ -67,7 +67,7 @@ void printsHelp() throws Exception {
     void deliversCleanOutput() throws Exception {
         MatcherAssert.assertThat(
             MainTest.exec("org.eolang.io.stdout", "Hi!"),
-            Matchers.equalTo(String.format("Hi!%s", System.lineSeparator()))
+            Matchers.equalTo(String.format("Hi!Dataization result:%ntrue%n"))
```

Comment:
@mximp in my opinion, this doesn't look like a clean output. It's confusing. Maybe this one would be better:

```
Hi!
---
true
```

I mean this:

```
"%n---%n%s",
```